data/water/flood-areas/index.html

```diff
@@ -33,45 +33,61 @@
-      premium rates for flood coverage offered through the NFIP.</p>
-    <p>The DFIRM Database presents the flood risk information depicted on the FIRM in a digital format suitable for use
-      in electronic mapping applications. The DFIRM database is a subset of the Digital FIS database that serves to
-      archive the information collected during the FIS.</p>
-    <p>Floodplains Zone information can be found at the <a href="http://msc.fema.gov/portal"> FEMA Map Service Center
+    
+    <p>Anywhere it rains it can flood. Mapped floodplains or “Special Flood Hazard Areas (SFHA’s)” represent areas in 
+        Utah that are at a higher risk of flooding. In these areas there is a 1-percent annual chance that flood waters 
+        will reach or surpass a base flood elevation in any given year. The 1-percent annual chance flood is also 
+        referred to as the base flood or 100-year flood and the SFHA is commonly referred to as the "100-year floodplain.”</p>
+
+    <p>The 1-percent-annual-chance floodplain (or SFHA) is the area where the National Flood Insurance Program's (NFIP's) 
+        floodplain management regulations must be enforced through flood hazard building codes and the area where the 
+        mandatory purchase of flood insurance applies.</p>
+
+    <p>This layer was created using criteria as defined by the Flood Insurance Rate Maps (FIRM) and produced by the 
+        Federal Emergency Management Agency (FEMA). These mapped zones are regulatory.  FIRMs are the basis for 
+        floodplain management, mitigation, and insurance activities for the National Flood Insurance Program (NFIP).</p>
+
+    <p>Insurance applications include enforcement of the mandatory purchase requirement of the Flood Disaster Protection Act, 
+        which "... requires the purchase of flood insurance by property owners who are being assisted by Federal programs 
+        or by Federally supervised, regulated or insured agencies or institutions in the acquisition or improvement of 
+        land facilities located or to be located in identified areas having special flood hazards," Section 2 (b) (4) of 
+        the Flood Disaster Protection Act of 1973. Special Flood Hazard Areas (SFHAs) identified on the map are the risk 
+        zones shown on the FIRMs and are the basis for the establishment of premium rates for flood coverage offered 
+        through the NFIP.</p>
+
+    <p>The Digital Flood Insurance rate Map (DFIRM) Database presents the flood risk information depicted on the FIRM in 
+        a digital format suitable for use in electronic mapping applications. The DFIRM database is a subset of the 
+        Digital Flood Insurance Study (FIS) database that serves to archive the information collected during the FIS.</p>
+
+    <p>Floodp Zone information can be found at the <a href="http://msc.fema.gov/portal"> FEMA Map Service Center
```

Comment:
Is Floodp a spelling error?
```suggestion
    <p>Floodp Zone information can be found at the <a href="https://msc.fema.gov/portal"> FEMA Map Service Center
```